components/fieldRouter/index.tsx

```diff
@@ -60,18 +61,24 @@ export default function FieldRouter({
   }
   if (children && fieldClass === 'group') {
     const [value, setValue] = valueHook(index ?? name);
-    const useChildValue = (key: string): [any, Function] => {
-      const valueObject = value !== null && typeof value === 'object' && !Array.isArray(value) ? value : { [key]: value };
-      const childValue = valueObject ? valueObject[key] : null;
-      const setChildValue = (newValue: string) => {
-        const newValueObject = {
-          ...valueObject,
-          [key]: newValue,
+    let useChildValue: (key: string) => [any, Function];
+
+    if (serializeData) {
+      useChildValue = (key: string): [any, Function] => {
```

Comment:
Could the use of `any` be narrowed down here?

Reply:
not easily. I tried and it quickly cascades across the codebase.